### DISCUSSION TITLE
Is Twind still maintained?

### DISCUSSION THREAD

**Discussion by alantriesagain** [tw-in-js/twind#495] [Q&A] [answered]
Hey @sastan! First of all, thanks for creating twind! Of all "Tailwind-in-JS" libs available twind is the best I found and the one I could make it integrate with web components without a building/bundling step - no vite or webpack. Now that it seems to be a great fit for a project I'm developing, I'm wondering about the long term viability of Twind?

I'm getting to know a bit about the code also in order to make some contributions in the future. 

Anyway, thank you very much for making Twind!

**Comment by tmikaeld** [comment]
Now that Deno officially abandoned Twind, [saying it's unmaintained for a year.](https://deno.com/blog/fresh-1.6#first-class-tailwind-css-plugin) I'd say it's about damn time to either archive the project or do something about it - like look for other maintainers (maybe even the sponsors) to take over the helm.

**Reply by rschristian** [reply]
A bit late, but let's not use such hostile language in regards to software offered for free. OSS authors owe you nothing, and I say this as someone who's [donated a bit less than $1k to the project](https://opencollective.com/twind).

Sascha hasn't been around on GH at all, I hope he's alright, but sometimes project authors just leave and that's all there is to it.

**Comment by alantriesagain** [comment, marked as answer]
Just a note for the ones looking for an alternative, UnoCSS is a project very well maintained: https://github.com/unocss/unocss
Twind is abandoned and only updates the sponsors (probably automatically). 

@rschristian although I agree with you, taking advantage of the OSS community by collecting passive income while doing nothing isn't cool.

**Reply by rschristian** [reply]
Who's being taken advantage of? They're donations, not forced payment.

A junior engineer in an entry-level position will make more in a month, after taxes, than this project has earned in total. Even if there's no further contributions to the project, the maintainer has still invested far more time than he'll ever be adequately paid for.

**Reply by alantriesagain** [reply]
Getting donation from a project where people depend on it in place of getting people to join as maintainer is taking advantage of the OSS community. There are people that integrated their code with twind and have the trouble to migrate to another solution. They are donations, yes. Taking advantage of the good faith of OSS developers isn't a crime, just a shame.

**Comment by gavoja** [comment]
In search of a worthy Twind replacement, I ended up writing my own library. I tried UnoCSS, but the runtime is not that lightweight and it does require some setup. I needed something that just works (also in Shadow DOM) for fast prototyping.

The project is called TWCSS. Give it a go: https://github.com/gavoja/twcss
